tests/funit-core/Test_AssertString.pf

```diff
@@ -180,8 +180,6 @@ contains
      call assertEqual( &
           & expected = "i= 1 f= F s=word x=  1.23", &
           & found    = "i= 1  f= F s=word x=  1.23", &
-!          & expected = "i= 1 f= F s=word x=  1.23", &
-!          & found    = “i= 1  f= F s=word x=  1.23", &
```

Comment:
This line contains non-ascii char for double quote.

Reply:
I should have read more carefully.  I'd not have had you bother if I'd realized that the lines were commented out.  (At least I now know why the compilers were not complaining!)

I'm going to approve, but will probably not bother rolling out a release.

Reply:
No problem, happy to help. :-)